tutorials/fundamentals/regexp-starts-with.md

```diff
@@ -0,0 +1,14 @@
+To check if a string starts with a regular expression, use the `startsWith()` function.
+Regular expressions begin and end with a `/` but can have different characters in between, like `^`.
+
+```javascript
+let reg = '/ab+c/';
+reg.startsWith('/'); // true
```

Comment:
What I was trying to get at with this tutorial is `/^a/.test('abc'); // true` to test whether a string starts with 'a'

More on `test()`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/test